cypress/integration/api.ts

```diff
@@ -4,27 +4,85 @@ describe('API component', () => {
     cy.redirectUnpkgToLocalDistFolder('elements');
   });
 
-  it('loads correctly', () => {
-    loadZoomApiPage();
+  describe('Api page', () => {
+    it('loads api page correctly', () => {
+      loadZoomApiPage();
+      cy.findByText('https://api.zoom.us/v2').should('exist');
+      cy.findByRole('heading', { name: /API Key/ }).should('exist');
+      cy.findByText('API Description').should('exist');
+    });
   });
 
-  it('navigates via table of contents', () => {
-    loadZoomApiPage();
-    cy.findByText('groups').click();
-    cy.findByText('Create a group').click();
-    cy.findByRole('heading', { name: /Create a group/i }).should('exist');
+  describe('Operation page', () => {
+    it('loads operation docs correctly', () => {
+      cy.visit('/zoom-api/operations/groupCreate');
+      cy.findByRole('heading', { name: /Create a group/i }).should('exist');
+      cy.findByRole('heading', { name: /Request/i }).should('exist');
+      cy.findByRole('heading', { name: /post \/groups/i }).should('exist');
+      cy.findByRole('heading', { name: /Response/i }).should('exist');
+    });
+
+    it('does not break on select dropdown', () => {
+      loadZoomApiPage();
+      cy.visit('/zoom-api/operations/groups');
+      cy.findByRole('button', { name: /request sample/i }).click();
+      cy.findByText('Obj-C').should('exist');
+    });
+
+    it('displays request sample correctly', () => {
+      cy.visit('/zoom-api/operations/userCreate');
+      cy.findByLabelText(
+        'curl --request POST \\ --url \'https://api.zoom.us/v2/users?access_token=\' \\ --header \'Content-Type: application/json\' \\ --data \'{ "action": "create", "user_info": { "email": "string", "first_name": "string", "last_name": "string", "password": "string", "type": 1 } }\'',
+      ).should('exist');
+    });
+
+    it('displays example correctly', () => {
+      cy.visit('/zoom-api/operations/userCreate');
+      cy.findByLabelText(
+        '{ "email": "string", "first_name": "string", "id": "string", "last_name": "string", "type": 1 }',
+      ).should('exist');
+    });
+
+    it('displays schema correctly', () => {
+      cy.visit('/zoom-api/operations/userCreate');
+      cy.findAllByText('action').should('exist');
+      cy.findAllByText('password').should('exist');
+    });
+
+    it('invokes TryIt request (receives network error)', () => {
+      cy.visit('/zoom-api/operations/userCreate');
+      cy.findByRole('button', { name: /Send Request/i }).click();
+      cy.findByText('Network Error occured.').should('exist');
```

Comment:
hm... Again, this is testing the copy. Maybe checking for `/error/i` would be enough?